.github/workflows/test.yml

```
@@ -0,0 +1,29 @@
+name: Run tests
```

I'd suggest the following name.  If the PR fails this action, it can be because the build failed or because tests failed.

```suggestion
name: Build and Run Tests
```

---

Sure, I don't mind but I left this off initially because we don't really run any build right now 🤔

---

Spoke about this offline, since this doesn't actually run a build today, going to keep as is.